src/assets/warning.vue

```diff
@@ -0,0 +1,6 @@
+<template>
+    <svg width="50" height="47" viewBox="0 0 112 101" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can be reduced to

`<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 112 101"><path fill="#DB2721" d="M55.896 71a5 5 0 1 0 0 10 5 5 0 0 0 0-10Zm53.35 7.35-40.25-70a15 15 0 0 0-26.2 0l-40 70a15 15 0 0 0 12.8 22.649h80.6a14.991 14.991 0 0 0 13.145-7.51 14.998 14.998 0 0 0-.095-15.14Zm-8.65 10a4.997 4.997 0 0 1-4.4 2.55h-80.6a5 5 0 0 1-4.4-2.55 5 5 0 0 1 0-5l40-70a5 5 0 0 1 8.9 0l40.25 70a5 5 0 0 1 .25 5.1v-.1ZM55.896 31a5 5 0 0 0-5 5v20a5 5 0 1 0 10 0V36a5 5 0 0 0-5-5Z"/></svg>`